lib/monitoring/metrics.rb

```diff
@@ -0,0 +1,51 @@
+require 'prometheus/client'
+require 'prometheus/client/data_stores/direct_file_store'
+
+module Monitoring
+  module Metrics
```

Comment:
I think it might be good to name this Prometheus so that its clear what flavour of metrics you're getting.